src/components/LazyLoad/index.js

```diff
@@ -20,6 +20,19 @@ export default class LazyLoad extends Component {
     offset: 1000
   }
 
+  state = {
+    lazyLoadCalls: 0
+  };
+
+  constructor(){
+    super(...arguments);
+    this.debug = debug('component:LazyLoad');
```

Comment:
Don't need to set it on `this` thats bad practice. It's a standalone library that should not be saved to a component.